### PR TITLE
Added gems devise and devise-i18n

### PR DESCRIPTION
**Pull request by ekatebenkova**
@XaoGao 

Complete,  check PR

### CODE REVIEW

File: app/models/user.rb [hunk 1]
```diff
@@ -0,0 +1,6 @@
+class User < ApplicationRecord
+  # Include default devise modules. Others available are:
```
Comment (XaoGao) [inline app/models/user.rb:2]: @ekatebenkova remove all commanets

File: config/routes.rb [hunk 2]
```diff
@@ -1,4 +1,5 @@
 Rails.application.routes.draw do
+  devise_for :users
   # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
```
Comment (XaoGao) [inline config/routes.rb:3]: @ekatebenkova remove comments

File: test/fixtures/users.yml [hunk 3]
```diff
@@ -0,0 +1,11 @@
+# Read about fixtures at https://api.rubyonrails.org/classes/ActiveRecord/FixtureSet.html
```
Comment (XaoGao) [inline test/fixtures/users.yml:1]: We will use Factory bot for generate test entities, remove this file

File: test/models/user_test.rb [hunk 4]
```diff
@@ -0,0 +1,7 @@
+require "test_helper"
```
Comment (XaoGao) [inline test/models/user_test.rb:1]: For test, we will use rspec, remove this file

**Review by XaoGao — COMMENTED** [review]
@ekatebenkova fix comments

**Comment by ekatebenkova** [comment]
@XaoGao fixed comments and deleted files users.yml, user_test.rb 
Complete, check PR

**Comment by realvint** [comment]
@ekatebenkova in Gemfile we use double quotes

**Review by XaoGao — APPROVED** [review]

**Pull request merged**